SAMtemplates/state_machines/main.yaml

```diff
@@ -53,6 +75,9 @@ Resources:
         AdditionalPolicies: !Join
           - ","
           - - Fn::ImportValue: !Sub ${StackName}:functions:${GetMyPrescriptionsFunctionName}:ExecuteLambdaPolicyArn
+            - Fn::ImportValue: !Sub ${StackName}:functions:${EnrichPrescriptionsFunctionName}:ExecuteLambdaPolicyArn
+            - !ImportValue psu:functions:psu-GetStatusUpdates:ExecuteLambdaPolicyArn
```

Comment:
can we be consistent (at least in each block) and either use Fn::ImportValue or !ImportValue for all (they do the same).

Same for !Ref - could use Fn::Ref

Reply:
Have changed to `Fn::ImportValue` for consistency in the block.